x-pack/plugins/apm/public/components/app/mobile/search_bar.tsx

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { QueryDslQueryContainer } from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+import {
+  EuiFlexGroup,
+  EuiFlexGroupProps,
+  EuiFlexItem,
+  EuiSpacer,
+} from '@elastic/eui';
+import React from 'react';
+import { useBreakpoints } from '../../../hooks/use_breakpoints';
+import { ApmDatePicker } from '../../shared/date_picker/apm_date_picker';
+import { KueryBar } from '../../shared/kuery_bar';
+import { TimeComparison } from '../../shared/time_comparison';
+import { TransactionTypeSelect } from '../../shared/transaction_type_select';
+import { MobileFilters } from './service_overview/filters';
+
+interface Props {
+  hidden?: boolean;
+  showKueryBar?: boolean;
+  showTimeComparison?: boolean;
+  showTransactionTypeSelector?: boolean;
+  showMobileFilters?: boolean;
+  kueryBarPlaceholder?: string;
+  kueryBarBoolFilter?: QueryDslQueryContainer[];
+}
+
+export function MobileSearchBar({
+  hidden = false,
+  showKueryBar = true,
+  showTimeComparison = false,
+  showTransactionTypeSelector = false,
+  showMobileFilters = false,
```

Comment:
Is this a use case for mobile? Maybe i'm missing something but in which cases we don't want to show the filters?

Reply:
for the time being service maps and alerts page

Reply:
Thank you, Didn't have those in my mental map 😄